helm/charts/determined/templates/db-deployment.yaml

```diff
@@ -19,7 +19,6 @@ spec:
         app: determined-db-{{ .Release.Name }}
```

Comment:
Does this not use persistent storage? if so where is the storage class specified?

Reply:
It is defined here: https://github.com/coreweave/coreweave_determined/blob/rahul/roles/helm/charts/determined/values.yaml#L61. The storage PVC YAML needs to be modified to derive the storage class from the nodeSelector and not this values.

Reply:
Not sure what you are answering to @rtalaricw but there is no PVC defined in `db-deployment.yaml`, the PVC is defined in https://github.com/coreweave/coreweave_determined/blob/rahul/roles/helm/charts/determined/templates/db-persistent-volume-claim.yaml and that one needs to be modified to automatically determine the PVC based on the `regionSelector` like: https://github.com/coreweave/kube-charts/blob/master/charts/filebrowser/templates/database-persistentvolumeclaim.yaml#L6

Reply:
Yes, I was referring to the deployment: 
`persistentVolumeClaim:
            claimName: determined-db-pvc-{{ .Release.Name }}
`
but I fixed this.